src/neptune/new/internal/operation.py

```diff
@@ -188,11 +224,19 @@ def to_dict(self) -> dict:
         ret = super().to_dict()
         ret["ext"] = self.ext
         ret["file_path"] = self.file_path
+        ret["clean_after_upload"] = self.clean_after_upload
         return ret
 
     @staticmethod
     def from_dict(data: dict) -> "UploadFile":
-        return UploadFile(data["path"], data["ext"], data["file_path"])
+        return UploadFile(data["path"], data["ext"], data["file_path"], data.get("clean_after_upload", False))
+
+    @staticmethod
+    def get_upload_path(attribute_path: List[str], extension: str, upload_path: Path):
+        now = datetime.now()
+        tmp_file_name = f"{'_'.join(attribute_path)}-{now.timestamp()}-{now}.{extension}"
+        tmp_file_name = tmp_file_name.replace(" ", "_").replace(":", ".")
```

Comment:
What about just proper `strftime()` instead of these bad looking replace calls?

Reply:
I've used to copy-paste this code over and over...
Yes, it's better idea ;)